src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/HealthCheckSpecification.groovy

```diff
@@ -20,6 +23,23 @@ class HealthCheckSpecification extends BaseSpecification {
         expect: "Kilda's health check request is successful"
         northbound.getHealthCheck().components["kafka"] == "operational"
 
+        and: "All zookeeper nodes are in expected state"
+        def zk = new ZooKeeper(zkConnectString, 5000, {})
+        def activeColor = getActiveNetworkColor(zk)
+        def zkAssertions = new SoftAssertions()
+        ["connecteddevices", "floodlightrouter", "flowhs", "isllatency", "nbworker", "network",
```

Comment:
Should we also check the `grpc, northbound and swmanager` components?

Reply:
there is no `state` for grpc and NB, but yes, we need to add swmanager here